schema/2.0.0/corpora_schema.md

```diff
@@ -26,16 +26,17 @@ This schema supports multiple assay types. Each assay takes the form of one or m
 The schema additionally describes how the dataset, genes, and cells are annotated to describe the biological and technical characteristics of the data.
 
 This document is organized by:
+
 * [General requirements](#general-requirements)
 * [`X` (Matrix layers)](#x-matrix-layers), which describe the data required for different assays
 * [`obs` (Cell metadata)](#obs-cell-metadata), which describe each cell in the dataset
-* [`var` (Gene metadata)](#var-gene-metadata), which describe each gene in the dataset
+* [`var`/`raw.var` (Gene metadata)](#var-gene-metadata), which describe each gene in the dataset
```

Comment:
The link is broken. Can you try:
`[`var` and `raw.var` (Gene Metadata)](#var-and-rawvar-gene-metadata])`